localization/ekf_localizer/include/ekf_localizer/ekf_localizer.hpp

```diff
@@ -203,7 +203,7 @@ class EKFLocalizer : public rclcpp::Node
   std::queue<PoseInfo> current_pose_info_queue_;      //!< @brief current measured pose
   geometry_msgs::msg::PoseStamped current_ekf_pose_;  //!< @brief current estimated pose
   geometry_msgs::msg::PoseStamped
-    current_ekf_pose_no_yawbias_;  //!< @brief current estimated pose w/o yaw bias
+    current_biased_ekf_pose_;  //!< @brief current estimated pose w/o yaw bias
```

Comment:
The comment should also be fixed. How about something like this?
```suggestion
    current_biased_ekf_pose_;  //!< @brief current estimated pose without yaw bias correction
```

Reply:
Looks good!